### PR TITLE
add GroupsUtil to API references

### PR DESCRIPTION
**Pull request by nratskevich**

### CODE REVIEW

File: docs/source/api_reference.rst [hunk 1]
```diff
@@ -36,6 +36,15 @@ FilesUtil
         :members:
         :show-inheritance:
 
+.. _GroupsUtil:
+
+GroupsUtil
+----------------
```
Comment (alexander-polovtcev) [inline docs/source/api_reference.rst]: If I remember correctly Sphinx doesn't like when the number of underscores does not match header's length. Did you try compiling it?

Comment (nratskevich) [reply docs/source/api_reference.rst]: [Sphinx documentation](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) says that number of underscores can be bigger than the header:
> Section headers (ref) are created by underlining (and optionally overlining) the section title with a punctuation character, at least as long as the text
But it looks bad, so I'll fix it

Comment (alexander-polovtcev) [reply docs/source/api_reference.rst]: maybe I'm wrong and IDEA complained about that, not Sphinx

Comment (genestack-solomatin) [reply docs/source/api_reference.rst]: Sphinx drops warnings to console during `make html` about such cases.

**Pull request merged**